routingtable/neighbor.go

```diff
@@ -22,6 +22,12 @@ type Neighbor struct {
 	// Peer is a route server client
 	RouteServerClient bool
 
+	// Peer is a route reflector client
```

Comment:
Golint demands a comment like: RouteReflectorClient indicates peer is a route reflector client